library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
> The release is going to be documented to the clients can choose if they can/want to upgrade or not

you lose users doing this, you give a bad experience, you show that you care less about the effort it takes them to upgrade _when you could avoid it_

> Effort in order to remove/include this initialise is minimal in case this needs to be added in the future.

I don't think you are considering the impact to users of a public library when they upgrade. Nobody wants compile errors when they upgrade a library version. Nobody.

> I cannot see a reason why we would need to add the initialisation in the future,

Really? Consider _any_ upfront work wanting to be done by a library, cache initialisation, prefetching, loading, clearing, sorting, optimising

> but this breaking change in my opinion has a minimal impact.

As soon as any change is *breaking* its bad enough to warrant the question of "how can we avoid breaking the users experience"